test/e2e_node/pod_resize_test.go

```diff
@@ -1131,6 +1131,24 @@ func doPodResizeTests() {
 				},
 			},
 		},
+		{
+			name: "Burstable QoS pod, one container with only memory requests - increase memory requests",
+			containers: []TestContainerInfo{
+				{
+					Name:      "c1",
+					Resources: &ContainerResources{MemReq: "200Mi"},
+				},
+			},
+			patchString: `{"spec":{"containers":[
+						{"name":"c1", "resources":{"requests":{"memory":"300Mi"}}}
+					]}}`,
+			expected: []TestContainerInfo{
+				{
+					Name:      "c1",
+					Resources: &ContainerResources{MemReq: "300Mi"},
+				},
+			},
+		},
```

Comment:
How about testing "CPU only" requests as well?

Reply:
Thank you for the suggestion. I added a similar test case for CPU. In this case, CPU limit is necessary due to the issue #126388.